src/core/server/app/router/client.ts

```diff
@@ -114,6 +115,10 @@ const clientHandler = ({
     locale = req.coral.tenant.locale;
   }
 
+  config.featureFlags =
+    req.coral.tenant?.featureFlags?.filter(validFeatureFlagsFilter(req.user)) ||
+    [];
+
```

Comment:
This assignment should be avoided, instead use a object spread in the following `res.render` call.

Reply:
Of course!